### DISCUSSION TITLE
Eror: runtime/eng/native/gen-buildsys.sh: line 111

### DISCUSSION THREAD

**Discussion by alekseykarpushka** [dotnet/runtime#48910] [Q&A]
OS AstraLinux
An error occurred during the runtime build:

> /opt/runtime/eng/native/gen-buildsys.sh: line 111: 34676 Killed                  $cmake_command --no-warn-unused-cli -G "$generator" "-DCMAKE_BUILD_TYPE=$buildtype" "-DCMAKE_INSTALL_PREFIX=$__CMakeBinDir" $cmake_extra_defines $__UnprocessedCMakeArgs "$1"
  Unable to find generated build files for "CoreCLR component" project!
/opt/runtime/src/coreclr/runtime.proj(45,5): error MSB3073: выход из команды ""/opt/runtime/src/coreclr/build-runtime.sh" -x64 -debug -os Linux" с кодом 1.
> Ошибка сборки.
/opt/runtime/src/coreclr/runtime.proj(45,5): error MSB3073: выход из команды ""/opt/runtime/src/coreclr/build-runtime.sh" -x64 -debug -os Linux" с кодом 1.
    Предупреждений: 0
    Ошибок: 1
Прошло времени 00:00:52.90
Build failed with exit code 1. Check errors above.

The installation was carried out according to the instructions attached in the file linux-requirements.md.
All the necessary packages have been installed.

**Comment by danmoseley** [comment]
Are there any earlier errors in the log?
